predicators/spot_utils/spot_utils.py

```diff
@@ -489,7 +489,7 @@ def graspController(self, objs: Sequence[Object], params: Array) -> None:
         if "_table" in objs[2].name:
             self.hand_movement(params[:3], keep_hand_pose=False, angle_45=True)
         self.arm_object_grasp(objs[1])
-        if not all(params[:3] == [0.0, 0.0, 0.0]):
+        if not np.allclose(params[:3], [0.0, 0.0, 0.0]):
```

Comment:
i'm pretty sure there was one more case of this, can you double check?

Reply:
That was already fixed last PR